Sources/ViewInspector/Inspector.swift

```diff
@@ -110,9 +117,8 @@ public struct Inspector {
     ]
 
     private static let sanitizeNamespaceRegex = {
-        // swiftlint:disable force_try
+        // swiftlint:disable:next force_try
```

Comment:
total NIT, if you inline this it's easier for IDE functions (like raising/lowering the line)
```
try! NSRegularExpression(pattern: sanitizeNamespacePatterns.joined(separator: "|")) // swiftlint:disable:this force_try
```